<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickIconDialog</class>
 <widget class="QDialog" name="PickIconDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pick Icon</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="buttonAdd">
     <property name="text">
      <string>Add icon</string>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonRemove">
     <property name="text">
      <string>Remove Icon</string>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QListWidget" name="iconsList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::SinglePass</enum>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="batchSize">
      <number>100</number>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PickIconDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PickIconDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
